superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -94,6 +96,10 @@ const RightMenu = ({
     state => state.dashboardInfo?.id,
   );
 
+  const [, setQuery] = useQueryParams({
```

Comment:
oh this is interesting, I hadn't seen this before. Does this bypass the variable and only create the function?

Reply:
no, this just ignores the value in the first position, but the function returns the whole thing.
Just a nice syntax sugar thing when destructuring

Reply:
yeah I like it too!